roles/openstack_cli/defaults/main.yml

```
@@ -27,6 +27,7 @@ openstack_cli_packages:
   - python3-openstackclient
   - python3-senlinclient
   - python3-swiftclient
+  - python3-osc-placement
```

lets order in alphabetic
```suggestion
  - python3-openstackclient
  - python3-osc-placement
  - python3-senlinclient
  - python3-swiftclient
```